client/src/components/home/SeriesCard.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import WishlistIcon from './WishlistIcon';
+import {
+  MDBCard,
+  MDBCardBody,
+  MDBCardTitle,
+  MDBCol,
+  MDBCardImage
+} from 'mdb-react-ui-kit';
+
+export default function SeriesCard(props) {
+  return (
+    <MDBCol>
+      <MDBCard>
+        <MDBCardImage src={props.series.poster_path} alt='...' position='top' />
```

Comment:
Should have a placeholder in case `poster_path` is falsy (`undefined` / `null` / `""` / etc.)
I already implemented a way to do so in my series details branch.